OpenUI5RelatedProjects.json

```diff
@@ -115,5 +115,23 @@
 			"owner": "jonathanbaker7",
 			"license": "Apache 2.0",
 			"type": "JS Library"
-	  }
+	  },
+		"openui5-password": {
+			"name": "OpenUI5 Password",
+			"description": "An OpenUI5 Control which checks your password strength and validates it.",
+			"githubLink": "https://github.com/mauriciolauffer/openui5-password",
+			"documentationLink": "https://github.com/mauriciolauffer/openui5-password",
+			"owner": "mauriciolauffer",
+			"license": "MIT",
+			"type": "JS Library"
+		},
+		"openui5-validator": {
+			"name": "OpenUI5 Generic App Testing",

```

Comment:
I think the title is wrong, there is a nother entry named "OpenUI5 generic app testing". Could you correct it to your name?

Reply:
Fixed and merged.